charts/library/common/templates/lib/cnpg/cluster/_backup.tpl

```diff
@@ -1,6 +1,16 @@
 {{- define "tc.v1.common.lib.cnpg.cluster.backup" -}}
   {{- $rootCtx := .rootCtx -}}
   {{- $objectData := .objectData -}}
+
+  {{- $compression := "gzip" -}}
```

Comment:
```suggestion
  {{- $compression := "bzip2" -}}
```

Imho, a standard of bzip2 gives better performance for the most frequent action (backup), as a trade off with longer restore time (the less frequent action).

But, more importantly, it also offers better compression. Which will save users costs.